integration-tests/actions/vrf/vrfv2plus/contract_steps.go

```diff
@@ -255,16 +219,11 @@ func CreateSubAndFindSubID(env *test_env.CLClusterTestEnv, chainID int64, coordi
 	if err != nil {
 		return nil, fmt.Errorf(vrfcommon.ErrGenericFormat, vrfcommon.ErrCreateVRFSubscription, err)
 	}
-	evmClient, err := env.GetEVMClient(chainID)
+	sethClient, err := env.GetSethClient(chainID)
 	if err != nil {
 		return nil, err
 	}
-	err = evmClient.WaitForEvents()
-	if err != nil {
-		return nil, fmt.Errorf(vrfcommon.ErrGenericFormat, vrfcommon.ErrWaitTXsComplete, err)
-	}
-
-	receipt, err := evmClient.GetTxReceipt(tx.Hash())
+	receipt, err := sethClient.Client.TransactionReceipt(context.Background(), tx.Hash())
```

Comment:
it should use the context provided from the test

Reply:
I'll fix it